packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -29,9 +36,32 @@ import {
   getAttributeByOriginAndCodeErrorMapper,
   getAttributesByNameErrorMapper,
 } from "../utilities/errorMappers.js";
+import { readModelServiceBuilderSQL } from "../services/readModelServiceSQL.js";
+
+const db = makeDrizzleConnection(config);
+const attributeReadModelServiceSQL = attributeReadModelServiceBuilderSQL(db);
+const tenantReadModelServiceSQL = tenantReadModelServiceBuilderSQL(db);
 
 const readModelRepository = ReadModelRepository.init(config);
-const readModelService = readModelServiceBuilder(readModelRepository);
+const oldReadModelService = readModelServiceBuilder(readModelRepository);
+const readModelServiceSQL = readModelServiceBuilderSQL(
+  db,
+  attributeReadModelServiceSQL,
+  tenantReadModelServiceSQL
+);
+
+const SQLFeatureFlagEnabled =
+  FeatureFlagSQL.safeParse(process.env).success && config.featureFlagSQL;
```

Comment:
`process.env` is already parsed at application startup in `config.ts` here: https://github.com/pagopa/interop-be-monorepo/pull/1651/files#diff-187b4509ffda18e0e4f4de2e2f95b7c61cba30b6d9cadc987bbe0a062af61054R24
So you could probably just directly use `config.featureFlagSQL` where needed

```suggestion
const SQLFeatureFlagEnabled = config.featureFlagSQL
```

Reply:
Sure, updated